docs/content/latest/api/ysql/exprs/window_functions/analyzing-a-normal-distribution/cr-bucket-using-width-bucket.md

```diff
@@ -2,7 +2,7 @@
 title: cr_bucket_using_width_bucket.sql
 linkTitle: cr_bucket_using_width_bucket.sql
 headerTitle: cr_bucket_using_width_bucket.sql
-description: Create the bucket function using the width_bucket built-in.
+description: Part of the code kit for the "Analyzing a normal distribution" section within the YSQL window fucntions documentation.
```

Comment:
Hmm...the description shouldn't be identical to other descriptions, like the previous one.

Reply:
Fixed globally by adding the name of the to-be-saved-as `.sql` script before the boilerplate in all of the `.md` files in this category—just as we agreed would be the plan.